config_metadata.yaml

```diff
@@ -0,0 +1,169 @@
+common:
```

Comment:
I don't know if it is necessary, given that the file content YAML structure is different from config.yaml. Alternatively, there could be a warning at the top of the file. Up to you guys.